testing/tests/DevExpress.ui.widgets.pivotGrid/sortable.tests.js

```diff
@@ -1247,9 +1268,11 @@ QUnit.module('sortable when source item is hidden', {
         $('<div id=\'first-group\' group=\'first\' class=\'group\'>')
             .append($container).appendTo($sortable);
 
-        $('<div id=\'sortable2\'><div id=\'second-group\' group=\'second\' class=\'group\' style=\'height: 150px\'><div class=\'test-container\'></div></div>')
+        $('<div id=\'sortable2\'><div id=\'second-group\' group=\'second\' class=\'group\'><div class=\'test-container\'></div></div>')
```

Comment:
Let's use the ``` ` ``` instead of the ``` ' ``` and make formatting more obvious:

```suggestion
        $(`
        <div id='sortable2'>
          <div id='second-group' group='second' class='group'>
            <div class='test-container'></div>
          </div>
        </div>
      `);
```